eli5/base.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+
+import attr
+from typing import Dict, List, Tuple, Union
```

Comment:
what do you think about swapping imports? I always think about typing as an stdlib module